.env.production

```
@@ -0,0 +1,3 @@
+GATSBY_TELEMETRY_DISABLED=1
+GA_TRACKING_ID="UA-123456789-0"
+API_ENDPOINT="https://pub-api.azurewebsites.net/api"
```

I believe this should be prefixed with `GATSBY_`?
```suggestion
GATSBY_API_ENDPOINT="https://pub-api.azurewebsites.net/api"
```

---

Hmmm I believe `API_ENDPOINT` also works based on the docs. https://www.gatsbyjs.org/docs/environment-variables/#client-side-javascript. What do you think? 

> For Project Env Vars that you want to access in client-side browser JavaScript, you can define an environment config file, .env.development and/or .env.production, in your root folder. Depending on your active environment, the correct one will be found and its values embedded as environment variables in the browser JavaScript.

> In addition to these Project Environment Variables defined in .env.* files, you could also define OS Env Vars. OS Env Vars which are prefixed with GATSBY_ will become available in browser JavaScript.

---

Hmm, I guess I misunderstood the docs. :+1: